app/api/auth.py

```diff
@@ -363,3 +365,38 @@ def decorated(*args, **kwargs):
 def environment_details():
     envdump = EnvironmentDump(include_config=False)
     return envdump.dump_environment()
+
+
+@ticket_blueprint.route('/orders/resend-email', methods=['POST'])
+@limiter.limit(
+    '5/minute', key_func=lambda: request.json['data']['order'], error_message='Limit for this action exceeded'
```

Comment:
Where is limit by user?

Reply:
@iamareebjamal modified order to user key.